README.md

```diff
@@ -136,7 +136,7 @@ Using this setting helps for massive models like BLOOM which require, or to avoi
 
 **Note that this option requires launching evaluation via `python main.py` rather than `accelerate launch main.py`.**
 
-To use `accelerate` with the `lm-eval` command, use 
+To use `accelerate` with the `lm-eval` command, use
```

Comment:
I think `lm-eval` command isn't yet mentioned elsewhere in readme?

Reply:
I added another line before this, in line 102 in the main README.md

```
Alternatively, you can use `lm-eval` instead of `python main.py` to call lm eval from anywhere.
```

Reply:
Ah, sorry, didn't catch that!